### PR TITLE
Adds TextureReplacedReplaced

### PR DESCRIPTION
**Pull request by MoreRobustThanYou**
Tired of not having this in CKAN!!

_I am not the author of this mod, I have linked to the author's page in the entry, however_

### CODE REVIEW

**Comment by politas** [comment]
Have you asked the author if they are okay with listing on CKAN?

**Comment by MoreRobustThanYou** [comment]
I'll ask (although by looking on the forum posts, it seemed like they WERE gonna do it, but never got around to it)

File: NetKAN/TextureReplacerReplaced.netkan [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+	"spec_version": "v1.4",
+	"name": "TextureReplacerReplaced",
+	"$kref": "#/ckan/github/HaArLiNsH/TextureReplacerReplaced",
+	"identifier": "TextureReplacerReplaced",
+	"release_status": "testing",
+	"ksp_version": "1.3.0",
```
Comment (Olympic1) [inline NetKAN/TextureReplacerReplaced.netkan]: The release includes a version file, could you delete `ksp_version` and add `"$vref": "#/ckan/ksp-avc",`

File: NetKAN/TextureReplacerReplaced.netkan [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	"author": ["shaw", "RangeMachine", "HaArLiNsH"],
+	"abstract": "Kerbal personalisation, IVA suits on Kerbin and texture replacement and improvements.",
+	"resources": {
+		"homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/161898",
+		"repository": "https://github.com/HaArLiNsH/TextureReplacerReplaced"
+	},
+	"provides": ["TextureReplacer"],
```
Comment (Olympic1) [inline NetKAN/TextureReplacerReplaced.netkan]: Also add `"conflicts": [ { "name": "TextureReplacer" } ],` so people cant install both mods, just to be sure.

**Comment by Olympic1** [comment]
Wanted to add this last week, but was still in pre-release. Good to see that there is a release now.

File: NetKAN/TextureReplacerReplaced.netkan (hunk 2 above)
Comment (MoreRobustThanYou) [reply NetKAN/TextureReplacerReplaced.netkan]: done

**Comment by Olympic1** [comment]
@politas do we want to filter all the KSP assemblies in the Plugins folder?

**Comment by HaArLiNsH** [comment]
Hey guys, I'm the author of this mod, you can put it on CKAN, but the last version of TRR (V0.5) is only for KSP 1.3.1. Users should be able to use both TR and TRR but I think TR won't work well in 1.3.1. This version is not a pre-release , it seems it has some minors visual bug in the menu but nothing critical.

Just one question, what do I need to do to maintain it ? I mean, what will I need to do for the next release to update it on CKAN ?

**Comment by MoreRobustThanYou** [comment]
@HaArLiNsH the easiest way to keep the versions up to date is with KSP-AVC and/or spacedock, I think.

**Comment by linuxgurugamer** [comment]
The author had contacted me about adding this. a few days ago.

I was just getting to it, glad to see it was done, but not happy it was submitted without any checking first.

**Comment by linuxgurugamer** [comment]
@Olympic1  @politas 
Either he needs to remove all those extraneous DLLs, or a filter needs to be added, otherwise problems will occur

**Comment by HaArLiNsH** [comment]
Yeah I feel a bit pushed, but its not a big deal.

You mean all the dll that I have in the sources ?  I wish I can get rid of them. I don't know why since KSP 1.3.1 when I build, It put all these dll. I just changed the references

**Comment by linuxgurugamer** [comment]
I've contacted the author via PM in the forums to get those DLLs removed.  I recommend NOT accepting this until that is done.  Among other things, those files are copyrighted

**Comment by HaArLiNsH** [comment]
Alright, I made a new version ( V0.5.1) without the .dll.  It was late when I made the last release, I should have noticed these files.

**Comment by HaArLiNsH** [comment]
I need to warn that using TR and TRR at the same time is NOT recommended. We need to have a check for this.

**Comment by Olympic1** [comment]
> Is there a guide somewhere so I can maintain myself the mod on CKAN ? I don't especially want to have to ask somebody to change this or this setting or if I must, at least I can ask it the right way :)

There are two ways to do this:
1. Place the `.netkan` in your own repo. This way you can edit the file directly, but it doesn't get checked by our bot (Jenkins) for problems.
2. Fork the NetKAN repo, make your changes on there and make a PR.

You can check our wiki on [how to add recommendations](https://github.com/KSP-CKAN/CKAN/wiki/Adding-a-mod-to-the-CKAN#dependencies-or-recommended-companion-mods).

> I need to warn that using TR and TRR at the same time is NOT recommended. We need to have a check for this.

This check is already in place in the PR.

**Comment by linuxgurugamer** [comment]
Verified the updated file
Thanks

**Comment by politas** [comment]
Your call, @Olympic1

**Comment by HaArLiNsH** [comment]
Thx for the informations @Olympic1 , I'll fork and make a PR, the homepage link is wrong :)

**Pull request merged**